### PR TITLE
Add selectFromConfigFile to MsalTestApp infra

### PR DESCRIPTION
**Pull request by amoyixi**
Add selectFromConfigFile functino to MsalTestApp infra in order to process LTW broker E2E test using MSA account

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -9,6 +9,7 @@ V.Next
 - [PATCH] Passing span context to AuthorizationActivity (#2209)
 - [MINOR] Bumping Moshi versions; force setting Okio version (#2210)
 - [PATCH] Fix to generate new Asymmetric Key (#2222)
+- [MINOR] Add selectFromConfigFile to MsalTestApp infra (#2235)
```
Comment (fadidurah) [inline changelog.txt]: I dont think we need changelog entry for this

**Review by fadidurah — APPROVED** [review]

File: changelog.txt (hunk 1 above)
Comment (somalaya) [reply changelog.txt]: +1

File: uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java [hunk 2]
```diff
@@ -147,6 +147,14 @@ public void selectFromAuthScheme(@NonNull final String text) throws UiObjectNotF
         authScheme.click();
     }
 
+    // select from Config File dropdown
```
Comment (somalaya) [inline uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/MsalTestApp.java]: nit : Select configuration to be used from dropdown.

**Review by somalaya — APPROVED** [review]
Approved with suggestions

**Pull request merged**